conference/react-js/src/containers/EventTalks/EventTalks.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import RaisedButton from 'material-ui/RaisedButton';
+
+import { myEvents } from '../../constants/route-url';
+import Talks from '../Talks';
+
+const columnsList = [
+  'name',
+  'title',
+  'topic',
+  'status',
+  'comment',
+];
+
+const EventTalks = ({
+  conference: {
+    id },
+  onClick,
+}) => (
+  <div className="tabs-layout">
+    <Link to={myEvents} >
+      <RaisedButton
+        label="Back"
+        primary
+      />
+    </Link>
+    <Talks
+      columns={columnsList}
+      onClick={onClick}
+      url={`/conference/${id}/talks`}
+    />
+  </div>
+);
+
+EventTalks.propTypes = { conference: PropTypes.shape({
```

Comment:
provide please tabulation. move `conference` to the new line. it is hard to read this block